internal/proto/data_coord.proto

```diff
@@ -250,7 +250,8 @@ message DataNodeInfo {
 
 message SegmentBinlogs {
   int64 segmentID = 1;
-  repeated FieldBinlog fieldBinlogs = 2;
+  int64 num_of_rows = 2;
```

Comment:
just add num_of_rows as third member, do not modify old members' order.

Reply:
fixed